flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -521,11 +525,11 @@ public void testCassandraTableSink() throws Exception {
             cmp.setField(0, o.getString(0));
             cmp.setField(1, o.getInt(2));
             cmp.setField(2, o.getInt(1));
-            Assert.assertTrue(
+            Assertions.assertTrue(
```

Comment:
I'm wondering whether may want to think about doing something stupid and add a `public class Assert extends Assertions{}`.

Reply:
Yes that was also my thought. At the very least it would be convenient to have for changing the asserts themselves (the optional message moved to the last param which is a nightmare to grep/replace).